src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -151,49 +151,64 @@
+                            library-schema request-refresh workspace-id storage-cost]}]
   (let [{:keys [owners]
-         {:keys [createdBy createdDate bucketName description workspace-attributes library-attributes]} :workspace} workspace]
+         {:keys [createdBy createdDate bucketName description workspace-attributes library-attributes]} :workspace} workspace
+        render-detail-box (fn [order title & children]
+                            [:div {:style {:flexBasis "50%" :order order}}
+                             (style/create-section-header title)
+                             children])]
     [:div {:style {:flex "1 1 auto" :overflow "hidden"}}
-     [:div {:style {:flex "1 1 auto" :display "flex"}}
-      [:div {:style {:flex "1 1 50%"}}
-       (style/create-section-header (str "Workspace Owner" (when (> (count owners) 1) "s")))
-       (style/create-paragraph
-         [:div {}
-          (interpose ", " owners)
-          (when owner?
-            [:span {}
-             " ("
-             (style/create-link {:text "Sharing..."
-                                 :onClick #(modal/push-modal
-                                            [AclEditor {:workspace-id workspace-id
-                                                        :request-refresh request-refresh}])})
-             ")"])])
-       (style/create-section-header "Created By")
-       (style/create-paragraph
-         [:div {} createdBy]
-         [:div {} (common/format-date createdDate)])]
-      [:div {:style {:flex "1 1 50%" :paddingLeft 10}}
-       (style/create-section-header "Google Bucket")
-       (style/create-paragraph
-         (case bucket-access?
-           nil [:div {:style {:position "absolute" :marginTop "-1.5em"}}
-                [comps/Spinner {:height "1.5ex"}]]
-           true (style/create-link {:text bucketName
-                                    :href (str moncommon/google-cloud-context bucketName "/")
-                                    :style {:color "-webkit-link" :textDecoration "underline"}
-                                    :title "Click to open the Google Cloud Storage browser for this bucket"
-                                    :target "_blank"})
-           false bucketName))
-       (style/create-section-header "Analysis Submissions")
-       (style/create-paragraph
-         (let [count-all (apply + (vals submissions-count))]
-           [:div {}
-            (str count-all " Submission" (when-not (= 1 count-all) "s"))
-            (when (pos? count-all)
-              [:ul {:style {:marginTop "0"}}
-               (for [[status subs] (sort submissions-count)]
-                 [:li {} (str subs " " status)])])]))]]
+     [:div {:style {:display "flex" :flexWrap "wrap"}}
+      (render-detail-box
+        1
+        (str "Workspace Owner" (when (> (count owners) 1) "s"))
+        (style/create-paragraph
+          [:div {}
+           (interpose ", " owners)
```

Comment:
Not sure if you're referring to the `str` or the `interpose`, but all of that is as I found it. I only extracted a function for the redundant parts of the detail flow box.